### PR TITLE
[FEAT] Iron Idol

### PR DESCRIPTION
**Pull request by Rofrtd**
# Description

Screenshot 2023-02-20 at 1 41 08 pm

Fixes #issue

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [x] Screenshot if it includes any UI changes
- [x] I have read the contributing guidelines and agree to the T&Cs
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/features/retreat/components/pirate/RetreatPirate.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+const bumpkin: Equipped = {
+  body: "Goblin Potion",
+  hair: "White Long Hair",
+  hat: "Pirate Hat",
+  shirt: "Fancy Top",
+  pants: "Pirate Pants",
+  tool: "Pirate Scimitar",
+  background: "Seashore Background",
+  shoes: "Black Farmer Boots",
+};
```
Comment (c-py) [inline src/features/retreat/components/pirate/RetreatPirate.tsx:23]: I think this NPC is different to the modal, might be worth spreading `...bumpkin` on `CloseButtonPanel`

**Comment by veenhdek** [comment]
Love the goblin pirate for TI items. Are these placeholder values for the iron idol? Imo we should encourage many people to mint. 200 seems low but also understand +1 is high so can we balance that to be like +.2 with a 1-2k supply. Or have it act similar to peeled potato where it's a 20% chance of +1?

**Review by c-py — APPROVED** [review]
Awesome works well for me - have left one comment around the NPC!

**Comment by TheDcol** [comment]
Similar concern to what Dekv said, also, I don't know how I feel about the art being behind the glass casing, feel like there are plenty glass casing nfts around and not sure how a bunch of them will suit to the isle. I am not an art person so bear that in mind :D

**Comment by TheDcol** [comment]
Similar concern to what Dekv said, also, I don't know how I feel about the art being behind the glass casing, feel like there are plenty glass casing nfts around and not sure how a bunch of them will suit to the isle. I am not an art person so bear that in mind :D

**Comment by mGranTY** [comment]
Supply should be higher and boost should be lower, most players have iron deficiency on their island, because of poor rng.

**Comment by Rofrtd** [comment]
> 

@veenhdek This PR is based on balanced Iron nodes, which is on our road map and will be happening soon. 
This item will take a while for people to mint, as it requires Bounties. 
The idea is to be a bit of an OP item with low supply.

@TheDcol we've tried without the case, but it didn't look good as the Idol is too thin.

**Comment by sabryder** [comment]
> Supply should be higher and boost should be lower, most players have iron deficiency on their island, because of poor rng.

Most of the community members AGREE with you on this👌🏻❤️

**Comment by 4dr3naline** [comment]
> Supply should be higher and boost should be lower, most players have iron deficiency on their island, because of poor rng.

Pick one square that shows two identical objects.❤️

**Comment by 4dr3naline** [comment]
> Supply should be higher and boost should be lower, most players have iron deficiency on their island, because of poor rng.

I totally agree🚀

**Pull request merged**